tests/product/tests/data/aws/aws_iam_test_cases.py

```diff
@@ -230,6 +233,35 @@
     "1.15 Permissions through groups: inline policy, expect: failed": cis_aws_iam_1_15_fail,
 }
 
+cis_aws_iam_1_16 = {
+    "1.16 Built-in EC2 policy, expect: passed": EksAwsServiceCase(
+        rule_tag=CIS_1_16,
+        case_identifier="AmazonEC2FullAccess",
+        expected=RULE_PASS_STATUS,
+    ),
+    "1.16 Attached AdministratorAccess Policy, expect: failed": EksAwsServiceCase(
+        rule_tag=CIS_1_16,
+        case_identifier="AdministratorAccess",
+        expected=RULE_FAIL_STATUS,
+    ),
+}
+
+cis_aws_iam_1_17 = {
+    "1.17 Attached support access policy, expect: passed": EksAwsServiceCase(
+        rule_tag=CIS_1_17,
+        case_identifier="AWSSupportAccess",
+        expected=RULE_PASS_STATUS,
+    ),
+}
+
+cis_aws_iam_1_20 = {
+    "1.20 Valid access analyzers in all regions, expect: passed": EksAwsServiceCase(
+        rule_tag=CIS_1_20,
+        case_identifier="account-access-analyzers",
+        expected=RULE_PASS_STATUS,
+    ),
+}
+
```

Comment:
@orestisfl, this test case (1.20) fails -> no evaluation found even after 10 minutes. Please check why it happens.

Reply:
Should be fixed by https://github.com/elastic/cloudbeat/pull/916 & https://github.com/elastic/csp-security-policies/pull/230

Reply:
@gurevichdmitry please check again